pkg/genericapiserver/storage_factory.go

```diff
@@ -268,3 +285,30 @@ func NewStorageCodec(storageMediaType string, ns runtime.StorageSerializer, stor
 	}
 	return runtime.NewCodec(encoder, decoder), nil
 }
+
+var specialDefaultResourcePrefixes = map[unversioned.GroupResource]string{
```

Comment:
How do we prove we don't regress on old clusters here?  Will 1.3 -> 1.4 deployment catch us.